dev-java/openjdk/openjdk-11.0.14_p9-r1.ebuild

```diff
@@ -36,9 +37,12 @@ SRC_URI="
 	!system-bootstrap? (
 		$(bootstrap_uri ppc64 ${PPC64_XPAK} big-endian)
 		$(bootstrap_uri x86 ${X86_XPAK})
+		riscv? ( https://github.com/Rabenda/gentoo-riscv-bootstarp-package/releases/download/bootstrap-openjdk-11/openjdk-bootstrap-11.0.15-riscv.tar.xz )
 	)
 "
 
+PATCHES=( "${FILESDIR}/openjdk-11.0.14-riscv.patch" )
```

Comment:
Sorry, but I can't accept that patch in FILESDIR!
It is whole 1.9MB which is super above our soft limit 10KB and hard limit 20KB.
To help the riscv project, I'm fine with using my devspace for it, so it is now hosted in the following [directory](https://dev.gentoo.org/~arthurzam/distfiles/dev-java/openjdk/)
Contact me if you want to upload a new version or what not 😄 

So now please download this patch in SRC_URI (preferably with `riscv? ( URL )`).

Also, as I don't know the origins of this patch, how tested and good are they, I prefer it to be conditionally applied on riscv only. Also, in case it won't apply in next version, we can drop it temporarily, unkeyword riscv and wait for new patch version. I promise to be on fast response.

Reply:
https://bugs.gentoo.org/782916#c3
Patch origin has been described

But it compiled failed in gentoo, I fixed it.

Reply:
OK, I see no issue with the patch origin.

But I do see issue with testing it on non-riscv arches. With all due respect, I don't want to check the full patch, understand if it might introduce issues for example ppc64. So to be extra safe, we can just apply it on riscv. As a result my logic is:

1. For amd64, ppc64, etc' - nothing changed, so we are safe and same as upstream (mostly)
2. For riscv, we currently have nothing at all. So even if there are issues, it is still better than nothing.

That is why I think conditional patching is better in this case.

Reply:
@Rabenda It would be good if you could also add some info about patch origin in the header or the patch